client/my-sites/domains/domain-management/list/site-domains.jsx

```diff
@@ -152,6 +156,10 @@ export class SiteDomains extends Component {
 
 		return (
 			<>
+				{ hasProductsList || <QueryProductsList /> }
+
+				{ this.isLoading() || <GoogleSaleBanner domains={ domains } /> }
```

Comment:
Can we keep the condition as an and operation? Feels weird to me to read it with or logic, plus we are not keeping consistency with the rest of the code 🙂 
```suggestion
				{ ! this.isLoading() && <GoogleSaleBanner domains={ domains } /> }
```

Reply:
I've forgot to mention the same change for: `{ hasProductsList || <QueryProductsList /> }` can we also change the logic to match the && pattern?

Reply:
It doesn't feel weird at all to me. I'd argue that the readability improves without the negation operator, which is why I added that in there. For now, I'll go with consistency.